assets/js/main.js

```diff
@@ -46,26 +46,66 @@ $(function(){
 
 });
 
-$('.secondary-sidenav-link').click(function(e) {
-  e.preventDefault();
-  var hashLocation = $(this).attr('href').split('#')[1]; // long url splitting
-  var anchor       = $('#' + hashLocation);
-  var headerOffset = 0;
-
-  /* :target already adds spacing to line up the page correctly
-   * prevent double space if current page target already = new target
-   */
-  if (!anchor.is(":target")) {
-    headerOffset = ($('.usa-site-header').first().outerHeight());
+/* Calculates what scrollTop should be in order to
+ * show an anchor properly under the header
+ * and lined up with the nav like the H1
+ */
+var calculateAnchorPosition = function (hash) {
+  var anchor        = $('#' + hash);
+  var topOffset     = 0;
+  var navPadding    = parseInt($('.sidenav').css('padding-top'), 10);
+  var anchorPadding = parseInt(anchor.css('padding-top'), 10);
+
+  if (anchor.length === 0) {
+    return topOffset;
+  }
+  
+  //start with the height of the header
+  topOffset = $('.usa-site-header').first().outerHeight();
+  //subtract the diffence in padding between nav top and anchor
+  topOffset = topOffset - (anchorPadding - navPadding);
+  
+  //anchor should now align with first item inside nav
+  return anchor.offset().top - topOffset;
+}
+
+$(function () {
+  var hash          = window.location.hash.substr(1);
+  var scrollTopPos  = calculateAnchorPosition(hash);
+
+  if (scrollTopPos > 0) {
+    setTimeout(function () {
```

Comment:
The normal behavior of anchors is to position themselves at the top of the browser window. Because the header is position: fixed, this means that the title of the section is by default covered by the header. This fixes that by setting scrollTop on page load when a user lands on a page with a hash on the url.